converters/tokenMetaData_test.go

```diff
@@ -0,0 +1,38 @@
+package converters
+
+import (
+	"testing"
+
+	"github.com/ElrondNetwork/elastic-indexer-go/data"
+	"github.com/ElrondNetwork/elastic-indexer-go/mock"
+	"github.com/ElrondNetwork/elrond-vm-common/data/esdt"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPrepareTokenMetaData(t *testing.T) {
+	t.Parallel()
+
+	require.Nil(t, PrepareTokenMetaData(nil, nil))
+	require.Nil(t, PrepareTokenMetaData(&mock.PubkeyConverterMock{}, nil))
+
+	require.Equal(t, &data.TokenMetaData{
```

Comment:
please add a supplemental variable expectedTokenMetaData. Also, please define a variable for the PrepareTokenMetaData call

Reply:
done.